### DISCUSSION TITLE
Is it possible to send an email that is not connected to Animal or Person?

### DISCUSSION THREAD

**Discussion by kernny** [sheltermanager/asm3#1363] [Q&A / Help] [answered]
Hi, just looking to see if there is a way to select an email template and send it out to people who are not in the system and not connected to an animal. 

The idea is to utilitse the email system in SM to send New Foster Carer Application form and information emails and other emails like that.

I thought the MAIL menu would be a good option but it's obviously for Mail Merge and that is not what I want to do. 

Would I have to generate a 'fake' person profile and send it from there each time or is there another, much cleverer way that I haven't found?

Thanks!

Kerny

**Comment by Tetzor** [comment, marked as answer]
Hi, if you have the persons name and email address, I'm not sure why you wouldn't create a person record. However you can send emails without a person record in the Document repository (ASM->Document repository).

Kind regards,

Jon
